docs/deployed-contracts/hoodi.md

```diff
@@ -0,0 +1,118 @@
+- OracleDaemonConfig: [`0x2a833402e3F46fFC1ecAb3598c599147a78731a9`](https://hoodi.cloud.blockscout.com/address/0x2a833402e3F46fFC1ecAb3598c599147a78731a9)
+- Legacy Oracle (Lido Oracle before V2, obsolete): [`0x5B70b650B7E14136eb141b5Bf46a52f962885752`](https://hoodi.cloud.blockscout.com/address/0x5B70b650B7E14136eb141b5Bf46a52f962885752) (proxy)
+
+## DAO contracts
+
+- Lido DAO (Kernel): [`0xA48DF029Fd2e5FCECB3886c5c2F60e3625A1E87d`](https://hoodi.cloud.blockscout.com/address/0xA48DF029Fd2e5FCECB3886c5c2F60e3625A1E87d) (proxy)
+- LDO token: [`0xEf2573966D009CcEA0Fc74451dee2193564198dc`](https://hoodi.cloud.blockscout.com/address/0xEf2573966D009CcEA0Fc74451dee2193564198dc)
+- Aragon Voting: [`0x49B3512c44891bef83F8967d075121Bd1b07a01B`](https://hoodi.cloud.blockscout.com/address/0x49B3512c44891bef83F8967d075121Bd1b07a01B) (proxy)
+- Aragon Token Manager: [`0x8ab4a56721Ad8e68c6Ad86F9D9929782A78E39E5`](https://hoodi.cloud.blockscout.com/address/0x8ab4a56721Ad8e68c6Ad86F9D9929782A78E39E5) (proxy)
+- Aragon Finance: [`0x254Ae22bEEba64127F0e59fe8593082F3cd13f6b`](https://hoodi.cloud.blockscout.com/address/0x254Ae22bEEba64127F0e59fe8593082F3cd13f6b) (proxy)
+- Aragon Agent: [`0x0534aA41907c9631fae990960bCC72d75fA7cfeD`](https://hoodi.cloud.blockscout.com/address/0x0534aA41907c9631fae990960bCC72d75fA7cfeD) (proxy)
+- Aragon ACL: [`0x78780e70Eae33e2935814a327f7dB6c01136cc62`](https://hoodi.cloud.blockscout.com/address/0x78780e70Eae33e2935814a327f7dB6c01136cc62) (proxy)
+- Voting Repo: [`0xc972Cdea5956482Ef35BF5852601dD458353cEbD`](https://hoodi.cloud.blockscout.com/address/0xc972Cdea5956482Ef35BF5852601dD458353cEbD) (proxy)
+- Token Manager Repo: [`0xCdE5696e83B1Fb6B8321e35361bB6e9A8bCbfb3f`](https://hoodi.cloud.blockscout.com/address/0xCdE5696e83B1Fb6B8321e35361bB6e9A8bCbfb3f) (proxy)
+- Finance Repo: [`0xa91aA04E0D6a06063d2E878309B60b723D75584d`](https://hoodi.cloud.blockscout.com/address/0xa91aA04E0D6a06063d2E878309B60b723D75584d) (proxy)
+- Agent Repo: [`0x7AA5670B2b4f6f0F7369F4F701C03ebFCe97d130`](https://hoodi.cloud.blockscout.com/address/0x7AA5670B2b4f6f0F7369F4F701C03ebFCe97d130) (proxy)
+- Lido App Repo: [`0xd3545AC0286A94970BacC41D3AF676b89606204F`](https://hoodi.cloud.blockscout.com/address/0xd3545AC0286A94970BacC41D3AF676b89606204F) (proxy)
+- Lido Oracle (Legacy Oracle) Repo: [`0x6E0997D68C1930a76413DE7da666D8A531eF1f9b`](https://hoodi.cloud.blockscout.com/address/0x6E0997D68C1930a76413DE7da666D8A531eF1f9b) (proxy)
+- Node Operators Registry Repo: [`0x52eff83071275341ef0A5A2cE48ee818Cef44c39`](https://hoodi.cloud.blockscout.com/address/0x52eff83071275341ef0A5A2cE48ee818Cef44c39) (proxy)
+- Simple DVT Repo: [`0x2b8B52A5e3485853aDccED669B1d0bbF31D40222`](https://hoodi.cloud.blockscout.com/address/0x2b8B52A5e3485853aDccED669B1d0bbF31D40222) (proxy)
+- Sandbox Repo: [`0x89D37eC788988e98BEceB32a8774394F1338B09C`](https://hoodi.cloud.blockscout.com/address/0x89D37eC788988e98BEceB32a8774394F1338B09C) (proxy)
```

Comment:
is not listed on deployed artifacts, but not a blocker

Reply:
Yeah, scratch deploy script didn't add them, had to fetch them manually through the ens resolver

Reply:
fyi: @arwer13 @tamtamchik

Reply:
Added except for the Sandbox Repo in https://github.com/lidofinance/core/pull/978/commits/863a3c0d5246d80ddd9bd78f0e90bcff7517f7ac